src/index.ts

```diff
@@ -0,0 +1,124 @@
+import { isAnyArray } from "is-any-array";
```

Comment:
this is the wrong style of quotes. can you add a `.prettierrc.json` file with our config and re-run prettier?

Reply:
Sorry I missed that!